### PR TITLE
feat(monster-game): add a sap the monster game

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Add a shoot the monster game

## Purpose
More games equals more fun

## Approach
The monster is approaching you so your gun has to fully charge so you can fire it. Shoot too soon, you lose. Shoot to late, the monster eats you.

Closes #63

### CODE REVIEW

**Review by dannysanchez559 — APPROVED** [review]
very interesting

**Review by JorgeLVilla — APPROVED** [review]
Dude always killing it!

File: components/killMonster/Gun.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    return (
+        <>
+            <Image 
+                source={charge} 
+                style={{ 
+                    position: "absolute", 
+                    bottom: chargePosition[0], 
+                    left: chargePosition[1],
+                    width: chargeSize[0],
+                    height: chargeSize[1],
```
Comment (kmausolf) [inline components/killMonster/Gun.js:19]: That's a neat trick, just resizing the image!

**Review by JayDampitan — APPROVED** [review]
Noice roach!

**Review by kmausolf — APPROVED** [review]
Pretty creative!  Nice work on this!

**Pull request merged**